packages/react-component-library/src/components/RangeSliderE/HandleE.tsx

```diff
@@ -41,7 +41,7 @@ export const HandleE: React.FC<HandleEProps> = ({
       aria-valuemin={min}
       aria-valuemax={max}
       aria-valuenow={value}
-      $left={`${percent}%`}
+      style={{ '--left': `${percent}%` }}
```

Comment:
As this is a new pattern I am wondering how it compares to passing `$left` as typically we may want to steer away from using a `style` prop as that is the plan for other components.

My point is if it is favourable over `$left` then there might be some situations we want to use `style` which is not obviously documented in the code.

Reply:
Hmm yeah it's only being used here over the transient prop for performance gain.

Using the css variables means we don't create new instances of the styled-component as the user drags the slider handle.

Maybe I should supplement it with a comment?

Reply:
Perhaps that and/or updating [Contributing docs](https://github.com/defencedigital/mod-uk-design-system/blob/master/docs/contributing.md#styled-components) as there appears to be nuance.